### ISSUE TITLE
How to release CPU memory cache in Libtorch JIT ?

### ISSUE THREAD

**Issue by w1d2s** [pytorch/pytorch#60847]
## ❓ Questions and Help
Hi every one, I would like to know to release CPU memory cache in Libtorch JIT? If there is no such way, can I set percentile of maximum memory used as cache ?  And I want to know if each torchscript::jit::module has its own memory cache , or all modules share one global memory cache ? Thanks.
### Please note that this issue tracker is not a help form and this issue will be closed.

We have a set of [listed resources available on the website](https://pytorch.org/resources). Our primary means of support is our discussion forum:

- [Discussion Forum](https://discuss.pytorch.org/)

**Comment by gmagogsfm** [comment]
Could you elaborate on "release CPU memory cache"?

As far as I know, JIT doesn't play any trick to try to manage CPU caches, so there is no point in releasing cache.

**Comment by w1d2s** [comment]
> Could you elaborate on "release CPU memory cache"?
> 
> As far as I know, JIT doesn't play any trick to try to manage CPU caches, so there is no point in releasing cache.

I made a test with an 140M parameters Transformer Encoder model，with different input size ( T , 1, 1320 ) where T = 300 / 1000 / 2000 / 3000 / 4000, and I test both with JIT's graph optimizer ON and OFF. The y-axis stands for free memory in MB.
![1625559691802-image](https://user-images.githubusercontent.com/7107330/124567601-be96e780-de76-11eb-9d20-e7b5d5870deb.png)

some key points in graph above:
1 : after T = 300 input inference, before T = 1000 input inference.
2 : after T = 1000 inference, before T = 2000 inference
3 : after T = 2000 inference, before T = 3000 inference
4 : after T = 3000 inference, before T = 4000 inference

my question is, from 1 to 4, the free memory is decreasing monotonically, in other words, with input tensor shape become larger, the memory occupation is becoming larger AFTER INFERENCE IS FINISHED, so I thought if there are some CPU memory cache strategies applied ? If not, why the memory usage AFTER EACH INFERENCE is increasing with input shape increasing ?  Thanks for attention.

I am using libtorch 1.9.1

**Comment by janeyx99** [comment]
TorchScript has been deprecated in favor of torch.export as of 2.11. https://pytorch.org/blog/pytorch-2-11-release-blog/ Closing this issue as we are not further developing on it--please ping if you believe this issue was erroneously closed.
